alphastats/plots/PlotUtils.py

```diff
@@ -42,9 +42,10 @@ class plotly_object(plotly.graph_objs._figure.Figure):
 class PlotUtils:
     @staticmethod
     def _update_colors_plotly(fig, color_dict):
-        # plotly doesnt allow to assign color to certain group
-        # update instead the figure in form of a dict
-        # color_dict with group_variable/legendgroup as key, and corresponding color as value
+        # TODO revisit this comment:
```

Comment:
https://plotly.com/python-api-reference/generated/generated/plotly.graph_objects.Figure.update_traces.html + selector to set individual color or something like:
plot.for_each_trace(lambda t: t.update(marker_color=color_dict.get(t.legendgroup))